src/numeric_fingerprint.js

```
@@ -20,16 +20,14 @@ function shortToArrayBuffer(number) {
+    var chunk = (hash[offset] * Math.pow(2, 32) +
+        hash[offset + 1] * Math.pow(2, 24) +
+        hash[offset + 2] * Math.pow(2, 16) +
+        hash[offset + 3] * Math.pow(2, 8) +
+        hash[offset + 4]) % 100000;
     var s = chunk.toString();
-    while (s.length < 5) {
-        s = '0' + s;
-    }
-    return s;
+
+    return s.padStart(5, '0')
```

ESLint

[error] Missing semicolon.

(semi)